embedded_postgres.go

```diff
@@ -200,9 +200,14 @@ func (ep *EmbeddedPostgres) Stop() error {
 
 func encodeOptions(port uint32, parameters map[string]string) string {
 	options := []string{fmt.Sprintf("-p %d", port)}
+	stringDelimitingChar := "'"
+	// CMD on Windows uses double quotes to delimit strings. It treats single quotes as regular characters.
+	if runtime.GOOS == "windows" {
```

Comment:
Why not just move to usign " on all platforms? If you try it do tests still all pass?

Reply:
I originally wanted to keep the existing behavior identical on unaffected platforms, but looks like the tests all pass when using ".

Reply:
Yeah, I reckon this is a better overall improvement though, nice job!